example/schema/places.js

```diff
@@ -0,0 +1,25 @@
+module.exports = {
+  id: {},
+  site: {
+    show: true
+  },
+  area: {
+    show: true
+  },
+  name: {
+    show: true
+  },
+  suitability: {
+    show: true,
+    sortable: false
+  },
+  holding: {
+    show: true,
+    sortable: false
+  },
+  nacwo: {
+    show: true,
+    accessor: 'nacwo.profile.name'
+  },
+  updatedAt: {}
+};
```

Comment:
Can you make this file a bit less ASL specific please?

Reply:
yeppo